Multisig/UI/Transaction/InitiateTransaction/ReviewSafeTransactionViewController/ReviewSafeTransactionViewController.swift

```diff
@@ -78,25 +78,39 @@ class ReviewSafeTransactionViewController: UIViewController {
         confirmButtonView.state = .normal
 
         confirmButtonView.onAction = { [weak self] in
+            self?.didConfirm()
+        }
+    }
+
+    func didConfirm() {
+        let keys = KeyInfo.owners(safe: self.safe)
+        if keys.isEmpty {
+            let addOwnerVC = AddOwnerFirstViewController()
+            addOwnerVC.onSuccess = { [weak self] in
+                self?.dismiss(animated: true)
+            }
+            let nav = ViewControllerFactory.modal(viewController: addOwnerVC)
+            presentModal(nav)
+            return
```

Comment:
I would continue the flow to ChooseOwnerKeyViewController  after adding the key instead of just return

Reply:
let me see how it works...

Reply:
it works great! 👏